indra/llcommon/tests/llleap_test.cpp

```diff
@@ -116,6 +115,7 @@ namespace tut
                    "except ImportError:\n"
                    // older llbase.llsd module
                    "    raise\n"
+                   "from llbase import llsd\n"
```

Comment:
That should only be necessary for someone who installed an older `llbase` and has not updated it since the version that brings in `llsd` implicitly. In fact a better change would be to remove the `try` / `except` surrounding `import llsd` - that was transitional.

Reply:
Resolved BUT!
Pulled another merge from main and had more conflicts so will need another review.